tools/StaticAnalysis/Exceptions/Az.OperationalInsights/BreakingChangeIssues.csv

```diff
@@ -0,0 +1,7 @@
+"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
+"Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.Clusters.GetAzureOperationalInsightsClusterCommand","Get-AzOperationalInsightsCluster","0","3000","The type of property 'KeyRsaSize' of type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSKeyVaultProperties' has changed from 'System.Int32' to 'System.Nullable`1[System.Int32]'.","Change the type of property 'KeyRsaSize' back to 'System.Int32'."
+"Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.Clusters.GetAzureOperationalInsightsClusterCommand","Get-AzOperationalInsightsCluster","0","1020","The cmdlet 'Get-AzOperationalInsightsCluster' no longer has output type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster'.","Make cmdlet 'Get-AzOperationalInsightsCluster' return type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster'."
+"Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.Clusters.NewAzureOperationalInsightsClusterCommand","New-AzOperationalInsightsCluster","0","1020","The cmdlet 'New-AzOperationalInsightsCluster' no longer has output type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster'.","Make cmdlet 'New-AzOperationalInsightsCluster' return type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster'."
+"Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.Clusters.RemoveAzureOperationalInsightsClusterCommand","Remove-AzOperationalInsightsCluster","0","2020","The cmdlet 'Remove-AzOperationalInsightsCluster' no longer supports the type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster' for parameter 'InputCluster'.","Change the type for parameter 'InputCluster' back to 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster'."
+"Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.Clusters.UpdateAzureOperationalInsightsClusterCommand","Update-AzOperationalInsightsCluster","0","1020","The cmdlet 'Update-AzOperationalInsightsCluster' no longer has output type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster'.","Make cmdlet 'Update-AzOperationalInsightsCluster' return type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster'."
+"Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.Clusters.UpdateAzureOperationalInsightsClusterCommand","Update-AzOperationalInsightsCluster","0","2020","The cmdlet 'Update-AzOperationalInsightsCluster' no longer supports the type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster' for parameter 'InputCluster'.","Change the type for parameter 'InputCluster' back to 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster'."
```

Comment:
```suggestion
"Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.Clusters.GetAzureOperationalInsightsClusterCommand","Get-AzOperationalInsightsCluster","0","1020","The cmdlet 'Get-AzOperationalInsightsCluster' no longer has output type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster'.","Make cmdlet 'Get-AzOperationalInsightsCluster' return type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster'."
"Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.Clusters.NewAzureOperationalInsightsClusterCommand","New-AzOperationalInsightsCluster","0","1020","The cmdlet 'New-AzOperationalInsightsCluster' no longer has output type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster'.","Make cmdlet 'New-AzOperationalInsightsCluster' return type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster'."
"Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.Clusters.RemoveAzureOperationalInsightsClusterCommand","Remove-AzOperationalInsightsCluster","0","2020","The cmdlet 'Remove-AzOperationalInsightsCluster' no longer supports the type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster' for parameter 'InputCluster'.","Change the type for parameter 'InputCluster' back to 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster'."
"Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.Clusters.UpdateAzureOperationalInsightsClusterCommand","Update-AzOperationalInsightsCluster","0","1020","The cmdlet 'Update-AzOperationalInsightsCluster' no longer has output type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster'.","Make cmdlet 'Update-AzOperationalInsightsCluster' return type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster'."
"Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.Clusters.UpdateAzureOperationalInsightsClusterCommand","Update-AzOperationalInsightsCluster","0","2020","The cmdlet 'Update-AzOperationalInsightsCluster' no longer supports the type 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster' for parameter 'InputCluster'.","Change the type for parameter 'InputCluster' back to 'Microsoft.Azure.Commands.OperationalInsights.Models.PSCluster'."
```
These breaking changes are introduced by the namespace change of PSCluster. Could you change the namespace of PSCluster back?